include/ideep/operators/conv.hpp

```diff
@@ -432,19 +697,118 @@ struct convolution_forward
                       const dims& padding_l,
                       const dims& padding_r,
                       int groups,
-                      const scale_t& src_scales = scale_t(),
-                      const scale_t& weights_scales = scale_t(),
-                      const scale_t& dst_scales = scale_t(),
+                      const scale_t& src_scales,
+                      const scale_t& weights_scales,
+                      const scale_t& dst_scales,
+                      const zero_point_t& src_zero_point,
+                      const zero_point_t& dst_zero_point,
                       const attr_t& attr = attr_t(),
                       algorithm aalgorithm = algorithm::convolution_direct,
                       prop_kind aprop_kind = prop_kind::forward,
                       const lowp_kind alowp_kind = u8s8,
                       const engine& aengine = engine::cpu_engine()) {
     static tensor dummy_bias;
-    compute_dispatch</*with_bias=*/false, plain_format>(
+    compute_dispatch</*with_bias=*/false, reorder_src, reorder_weight>(
         src, weights, dummy_bias, dst_dims, dst, strides, dilates,
         padding_l, padding_r, groups, src_scales, weights_scales, dst_scales,
-        zero_point_t(), zero_point_t(), attr, aalgorithm, aprop_kind, alowp_kind, aengine);
+        src_zero_point, dst_zero_point, attr, aalgorithm, aprop_kind, alowp_kind, aengine);
+  }
+
+  // Conv computation with pre-prepared params, with bias
+  //   fast_params: pd, primitive, groups ...
+  // reorder as true means reorder needed for src/weight/bias/dst as adapt
```

Comment:
```suggestion
  // reorder as true means reorder might be needed for src/weight/bias/dst as adapt
```

Reply:
Good catch, I will update all the comments accordingly.